python/feathub/processors/processor.py

```diff
@@ -24,10 +26,59 @@
     ProcessorType,
     PROCESSOR_TYPE_CONFIG,
 )
-from feathub.table.table import Table
-from feathub.table.table_descriptor import TableDescriptor
 from feathub.processors.processor_job import ProcessorJob
 from feathub.registries.registry import Registry
+from feathub.table.table import Table
+from feathub.table.table_descriptor import TableDescriptor
+
+
+class MaterializeJobDescriptor:
```

Comment:
It sounds better for me if we rename this class as `ProcessorJobDescriptor`, and move this class to an individual file or `processor_job.py`.